store/store.go

```diff
@@ -382,7 +382,7 @@ type WebhookStore interface {
 	GetIncomingList(offset, limit int) StoreChannel
 	GetIncomingByTeam(teamId string, offset, limit int) StoreChannel
 	UpdateIncoming(webhook *model.IncomingWebhook) (*model.IncomingWebhook, *model.AppError)
-	GetIncomingByChannel(channelId string) StoreChannel
+        GetIncomingByChannel(channelId string) ([]*model.IncomingWebhook, *model.AppError)
```

Comment:
Looks like build is failing due to formatting issues. Running `gofmt` should fix it. It's seeing spaces but want tab-spacing
``` 
GetIncomingList(offset, limit int) StoreChannel

 	GetIncomingByTeam(teamId string, offset, limit int) StoreChannel

 	UpdateIncoming(webhook *model.IncomingWebhook) (*model.IncomingWebhook, *model.AppError)

-        GetIncomingByChannel(channelId string) ([]*model.IncomingWebhook, *model.AppError)

+	GetIncomingByChannel(channelId string) ([]*model.IncomingWebhook, *model.AppError)

 	DeleteIncoming(webhookId string, time int64) StoreChannel

 	PermanentDeleteIncomingByChannel(channelId string) StoreChannel

 	PermanentDeleteIncomingByUser(userId string) StoreChannel

gofmt failure

Makefile:296: recipe for target 'gofmt' failed
```

Reply:
@s4kh yes, probably here would help some kind of configuration on your editor (is super common when you program in go to use gofmt or goimports (which call gofmt under the hood) automatically every time you write a go file. That is supported by the most common code editors like Vim, Emacs, Sublime, VsCode, Atom...